jberet-core/src/main/java/org/jberet/repository/JdbcRepository.java

```diff
@@ -1237,4 +1249,19 @@ private static String addPrefixSuffix(final String input,
         matcher.appendTail(sb);
         return sb.toString();
     }
+
+    /*
+    This method will take the raw value given in jberet.properties file such as
+    for ex. ${ENV_VAR_NAME:default_value} and return the value for the corresponding
+    Environment variable name given (eg. ENV_VAR_NAME from example) if it exists or
+    the default value (e.g default_value from example)
+     */
+    private String parseValuefromEnvVariables(String rawValue){
+        String[] valueList = rawValue.split(":",2);
+        String valueFromEnvVariable = System.getenv(valueList[0].replace("${","").trim());
+        if (valueFromEnvVariable == null){
+            return valueList[1].replace("}","").trim();
```

Comment:
val[1] can be non-existent, as in `${abc}`. so check for vals length is needed to avoid `ArrayOutOfBoundException`

Reply:
`getenv()` requires `RuntimePermission` and could throw `SecurityException`, so we should instead use `org.wildfly.security.manager.WildFlySecurityManager#getEnvPropertyPrivileged`